src/app/bitstream-page/edit-bitstream-page/edit-bitstream-page.component.ts

```diff
@@ -701,6 +721,11 @@ export class EditBitstreamPageComponent implements OnInit, OnDestroy {
     } else {
       bitstream$ = observableOf(this.bitstream);
     }
+    if (isAlternativeContent) {
+      updatedBitstream.setMetadata('dc.type', undefined, ...['alternative content']);
```

Comment:
I don't like how this value of `alternative content` has to be hardcoded in many areas of this PR.  It'd be better to use a single constant that could be referenced in all areas.  Since this is a hardcoded value, it also could be shortened to something like `alt-content`

Alternatively, this value could be moved to the backend as a constant (but that might require a new endpoint of some sort to tell the backend when a bitstream should be flagged as "alternative content")